packages/compiler/test/shadow_css/keyframes_spec.ts

```diff
@@ -0,0 +1,510 @@
+            animation: 3s 'foo\\\\', 3.5s "b\\\\\\"ar", 3.7s 'ba\\'\\"\\'r';
+            animation: 4s 'fo\\'o', 4.5s 'b\\"ar\\"', 4.7s "baz\\'";
+        }
+
+        @keyframes 'f"oo' {}
+        @keyframes 'fo""o' {}
+        @keyframes 'foo\\\\' {}
+        @keyframes 'fo\\'o' {}
+        @keyframes 'ba\\'r' {}
+        @keyframes 'b\\\\"ar' {}
+        @keyframes 'b\\\\\\"ar' {}
+        @keyframes 'b"ar"' {}
+        @keyframes 'ba\\'\\"\\'r' {}
+        `;
+       const result = s(css, 'host-a');
+       expect(result).toContain(`@keyframes 'host-a_f"oo' {}`);
+       expect(result).toContain(`@keyframes 'host-a_fo""o' {}`);
+       expect(result).toContain('@keyframes \'host-a_foo\\\\\' {}');
+       expect(result).toContain('@keyframes \'host-a_fo\\\'o\' {}');
+       expect(result).toContain('@keyframes \'host-a_ba\\\'r\' {}');
+       expect(result).toContain(`@keyframes 'host-a_b\\\\"ar' {}`);
+       expect(result).toContain(`@keyframes 'host-a_b\\\\\\"ar' {}`);
+       expect(result).toContain(`@keyframes 'host-a_b"ar"' {}`);
+       expect(result).toContain(`@keyframes 'host-a_ba\\'\\"\\'r' {}`);
+       expect(result).toContain(`animation: 1s 'host-a_f\\"oo', 1.5s "host-a_ba\\'r";`);
+       expect(result).toContain(`animation: 2s "host-a_fo\\"\\"o", 2.5s 'host-a_b\\\\"ar';`);
+       expect(result).toContain(
+           `animation: 3s 'host-a_foo\\\\', 3.5s "host-a_b\\\\\\"ar", 3.7s 'host-a_ba\\'\\"\\'r';`);
+       expect(result).toContain(
+           `animation: 4s 'host-a_fo\\'o', 4.5s 'host-a_b\\"ar\\"', 4.7s "baz\\'";`);
+     });
+
+  it('should handle the usage of commas inside quotes', () => {
+    const css = `
+        div {
+            animation: 3s 'bar,, baz';
+        }
+
+        p {
+            animation-name: "bar,, baz", foo,'ease, linear , inherit', bar;
+        }
+
+        @keyframes 'foo' {}
+        @keyframes 'bar,, baz' {}
+        @keyframes 'ease, linear , inherit' {}
+        `;
+    const result = s(css, 'host-a');
+    expect(result).toContain('@keyframes \'host-a_bar,, baz\' {}');
+    expect(result).toContain('animation: 3s \'host-a_bar,, baz\';');
+    expect(result).toContain(
+        `animation-name: "host-a_bar,, baz", host-a_foo,'host-a_ease, linear , inherit', bar;`);
+  });
+
+  it('should not ignore animation keywords when they are inside quotes', () => {
+    const css = `
+        div {
+            animation: 3s 'unset';
+        }
+
+        button {
+            animation: 5s "forwards" 1s forwards;
+        }
+
+        @keyframes unset {}
+        @keyframes forwards {}
+        `;
+    const result = s(css, 'host-a');
+    expect(result).toContain('@keyframes host-a_unset {}');
+    expect(result).toContain('@keyframes host-a_forwards {}');
+    expect(result).toContain('animation: 3s \'host-a_unset\';');
+    expect(result).toContain('animation: 5s "host-a_forwards" 1s forwards;');
+  });
+
+  it('should handle correctly css functions', () => {
```

Comment:
nit: this reads better to me but I'm not native English, so you're likely the expert here.

```suggestion
  it('should handle css functions correctly', () => {
```

Reply:
I am not a native speaker either unfortunately :sweat_smile: :sweat: 

This is pretty minor and I am totally fine with it either way :slightly_smiling_face::+1: